src/libswid.h

```diff
@@ -0,0 +1,80 @@
+#ifndef LIBSWID_H
+#define LIBSWID_H
+
+#include <stddef.h>
+
+#include "swid-common.h"
+
+
+struct SWIDTagIO;
+typedef struct SWIDTagIO * SWIDIOHandle;
+
+
+struct SWIDEntity;
+typedef struct SWIDEntity * SWIDEntityHandle;
+
+
+struct SWIDLink;
+typedef struct SWIDLink SWIDLinkHandle;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the unused declaration of type "SWIDLinkHandle". ([cpp:S897](https://sonarcloud.io/organizations/openscap/rules#rule_key=cpp%3AS897))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=openscap-libswid&pullRequest=9&issues=AWODAzRQdIhI-sZE26p2&open=AWODAzRQdIhI-sZE26p2)

[ISSUE_KEY]: AWODAzRQdIhI-sZE26p2